src/test/regress/sql/join_hash.sql

```diff
@@ -593,16 +593,19 @@ ROLLBACK;
 -- re-use of the inner hash table across rescans.
 begin;
 set local enable_hashjoin = on;
+-- GPDB: Join with the replicated table to avoid passing the parameter through motion.
+create table int4_tbl_repl (like int4_tbl) distributed replicated;
+insert into int4_tbl_repl select * from int4_tbl;
```

Comment:
I think it is more logical to do it this way:
```suggestion
create table tenk1_repl (like tenk1) distributed replicated;
insert into tenk1_repl select * from tenk1;
```